filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -557,3 +557,12 @@ filebeat.inputs:
   # Configure stream to filter to a specific stream: stdout, stderr or all (default)
   #stream: all
 
+#------------------------------ Journald input --------------------------------
```

Comment:
@kvch  Does this config need to be added here? Journald input is still experimental and before there was no mention to it in the reference configuration, so I'm not sure whether it should be added or not.

Reply:
Yes, it's ok if you add it. Please add a sentence about the input, including that is experimental.

Reply:
Done! (changes will be pushed soon).

Quick questions:
1. Do I need to add the full set of configuration? 
2. Should I link somehow to the [documentation](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-input-journald.html)

I feel that compared to all other inputs journald is missing a proper documentation in the reference config. Or is it like that because it's experimental?

Reply:
It is good as it is. I have planned a few breaking changes in filtering. When that is implemented, I will write a more detailed documentation page for it.